protecode/client.py

```diff
@@ -95,10 +96,24 @@ def scans(self, product_id: int):
 
 
 class ProtecodeApi:
-    def __init__(self, api_routes, basic_credentials, tls_verify=True):
+    def __init__(
+        self,
+        api_routes,
+        credentials,
+        tls_verify=True,
+    ):
         self._routes = not_none(api_routes)
-        self._credentials = not_none(basic_credentials)
-        self._auth = (basic_credentials.username(), basic_credentials.passwd())
+        self._credentials = not_none(credentials)
+
+        # if self._auth is ProtecodeClientAuth.BEARER_TOKEN:
```

Comment:
please do not check-in commented-out code-blocks, unless there is a good reason to do so